### DISCUSSION TITLE
增加术语库替换

### DISCUSSION THREAD

**Discussion by chNg1o** [immersive-translate/immersive-translate#1046] [Ideas]
### 新功能 | New Feature

1.自定义术语库
2.从术语库中替换翻译词汇

### 为什么您想要添加这些功能 | Why you want to add these features

1.更精确结合背景知识翻译
2.彩云小译好像有？

### 提议 | Proposal

可以在翻译渲染之后再从术语库中替换，更容易实现

**Comment by theowenyoung** [comment]
#91 相关讨论
